docs/architecture/grpc-for-wcf-developers/wcf-bindings.md

```diff
@@ -1,11 +1,37 @@
 ---
```

Comment:
This file is also included in #14303. What's the correct chapter for this? We've already done some edits on that other PR.

Reply:
Yeah, it shouldn't be in this one.

Reply:
Removed